linode/networkipassignment/framework_resource.go

```diff
@@ -0,0 +1,154 @@
+package networkipassignment
```

Comment:
Could this package be renamed to `networkingipassignment` to be consistent with the other package names?